assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -94,8 +94,10 @@ che.workspace.terminal_linux_arm7=${che.home}/lib/linux_arm7/terminal
 
 che.workspace.exec_linux_amd64=${che.home}/lib/linux_amd64/exec
 
-# RAM default for each machine that has no RAM settings in environment.
-che.workspace.default_memory_mb=1024
+# RAM limit default for each machine that has no RAM settings in environment.
```

Comment:
is this a property rename?

Reply:
yes

Reply:
So we need.
1. Add an alias for backward compatibility. https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che_aliases.properties#L13
2. Replace all existed values including ```CHE_WORKSPACE_DEFAULT__MEMORY__MB``` .   https://github.com/eclipse/che/blob/master/dockerfiles/init/manifests/che.env#L95
3. Write some release notes to explain what is going on.

Reply:
4. Explain why you need 2GB instead of all 1GB

Reply:
i will check what is needed regarding 1,2
3. release notes in the PR itself right ? or the documentation ?
4. I dont actually need 2GB , i just wanted default requested memory be less than the limit,
but if the limit stays 1GB and requested is lets say 512Mb that seems too little no ? would you suggest request to be the same as the limit by default ?

Reply:
3. PR. for the docs that are another story. I think we need @eivantsov opinion here.
4. Analyzing.

Reply:
4. For me, 512mb as request and 1gb as limit looks OK. 
It should not change the current behavior, am I right?